modules/stats.php

```diff
@@ -243,17 +231,12 @@ function stats_add_shutdown_action() {
  * @return void
  */
 function stats_footer() {
-	global $rendered_stats_footer;
-
-	if ( ! $rendered_stats_footer ) {
-		$data = stats_build_view_data();
-		if ( Jetpack_AMP_Support::is_amp_request() ) {
-			stats_render_amp_footer( $data );
-		} else {
-			stats_render_footer( $data );
-		}
-		$rendered_stats_footer = true;
-	}
+    $data = stats_build_view_data();
```

Comment:
Could you switch from spaces to tabs in here, to respect WordPress' coding standards? Setting your IDE to do that for you may help:
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#use-php-codesniffer-and-eslint-to-make-sure-your-code-respects-coding-standards

Reply:
Sorry, I've fixed that now.